andino_hardware/README.md

```diff
@@ -190,6 +190,91 @@ source install/setup.bash
 After this, you are good to go and use the robot!
```

Comment:
> After this, you are good to go and use the robot!

Is this still true or the steps below are mandatory to make it work?

If the steps below are indeed mandatory to make it work, I suggest changing the order here and placing it before the robot workspace creation steps.

Reply:
Nice catch. Done!